### PR TITLE
Use GitHub CLI

### PR DESCRIPTION
**Pull request by szepeviktor**
Every GHA image includes GitHub CLI.
So use it instead of a generic HTTP request.

### CODE REVIEW

**Comment by szepeviktor** [comment]
@pedro-mendonca Would you like me to replace the next `curl` usage too?

File: .github/workflows/update-build-files.yml [hunk 1]
```diff
@@ -103,21 +103,22 @@ jobs:
 
       - name: Check if Pull Request exists
         id: check_pr
+        env:
+          GH_TOKEN: "${{ github.token }}"
         run: |
-          BRANCH=main
-          RESPONSE=$(curl -X GET -H "Authorization: token $TOKEN" "https://api.github.com/repos/$REPO/pulls?head=$OWNER:english")
-          PR_NUMBER=$(echo "$RESPONSE" | jq -r '.[].number')
+          PR_NUMBER="$(gh pr list --head "${OWNER}:english" --json "number" --jq '.[0]."number"')"
```
Comment (szepeviktor) [inline .github/workflows/update-build-files.yml:109]: Adding `--repo "${REPO}"` is useless as it is the current repository.

**Comment by szepeviktor** [comment]
> Would you like me to replace the next `curl` usage too?

Done 🍏

**Comment by pedro-mendonca** [comment]
The GitHub CLI looks so much cleaner, thanks!

**Comment by szepeviktor** [comment]
You find all my brains in https://github.com/szepeviktor/byte-level-care/tree/master/.github/workflows

**Comment by pedro-mendonca** [comment]
It misses a checkout, but will add it in a further PR.
Thanks @szepeviktor

**Comment by szepeviktor** [comment]
> It misses a checkout

You are right, Pedro.
I am totally blind to that!

**Comment by pedro-mendonca** [comment]
It's fixed in #19

**Pull request merged**